src/pl/devsmentoring/java/Module5/Exercise13.java

```diff
@@ -0,0 +1,32 @@
+package pl.devsmentoring.java.Module5;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Exercise13 {
+
+    // Zad. 13 (T)
+// Napisz program, który będzie pobierał od użytkownika dowolne wyrazy,
+// dopóki nie wprowadzi on słowa “STOP”. Po podaniu każdego ze słów, ma
+// być ono umieszczane w obiekcie typu ArrayList. Po tak przeprowadzonym
+// procesie umieszczania kolejnych elementów, program ma wyświetlać długość
+// utworzonej list wraz z informacją, jaki jest jej pierwszy, ostatni i środkowy element.
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        List<String> newList = new ArrayList<>();
+
+        do {
+            System.out.println("Please enter any word: ");
+            newList.add(scanner.nextLine());
```

Comment:
if user passes "STOP" it will be added to the list. You have to check word before adding it to the list.